### PR TITLE
Docs: fleshing out Getting started overview.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: fleshing out Getting started overview. Closes cksource/ckeditor5-commercial#5936

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
...
 
-	CKEditor&nbsp;5 offers {@link getting-started/legacy-getting-started/predefined-builds predefined builds} that expose a rich JavaScript API, which you can use to {@link getting-started/legacy-getting-started/editor-lifecycle#creating-an-editor-with-create create editors} and {@link getting-started/getting-and-setting-data#setting-the-editor-data-with-setdata control them}.
+If you came here looking for a guide on how to suppress the branding logo, check the dedicated {@link support/managing-ckeditor-logo branding} guide.
 
-### Compatibility with CSS frameworks
+## Legacy installation methods
 
-CKEditor&nbsp;5 is compatible with popular CSS frameworks such as [Bootstrap](https://getbootstrap.com/) or [Foundation](https://get.foundation/). Such integrations, however, often require additional changes and adjustments that we have gathered {@link getting-started/integrations/css in this guide}.
+Before version 4x.x.x, CKEditor&nbsp;5 used different installation methods. If for whatever reason you wish to use these older versions &ndash; while it is not advised &ndash; you can have a look at the {@link getting-started/legacy-getting-started/quick-start legacy installation guides}.
```
Comment (gorzelinski) [inline docs/getting-started/index.md:65]: I guess there should be commas:
> If, for whatever reason, you wish to use these older versions...

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
...
-3. **If none exists, integrate CKEditor&nbsp;5 with your framework by yourself.**
+CKEditor&nbsp;5 offers {@link getting-started/legacy-getting-started/predefined-builds predefined builds} that expose a rich JavaScript API, which you can use to {@link getting-started/legacy-getting-started/editor-lifecycle#creating-an-editor-with-create create editors} and {@link getting-started/getting-and-setting-data#setting-the-editor-data-with-setdata control them}. -->
+
+## Configuring CKEditor&nbsp;5
+
+Once you install your copy of CKEditor&nbsp;5, take some time to {@link getting-started/setup/configuration configure it} before first use. Set up data saving methods, editor toolbars and UI.
+
+You may also take the {@link tutorials/crash-course/editor step-by-step tutorial} that will guide you through the installation and configuration of the editor.
+
+## Want to remove the "Powered by CKEditor" logo?
```
Comment (gorzelinski) [inline docs/getting-started/index.md:59]: I get that people need such a how-to. But I wonder if it's so important that we need to put it in the overview.

**Review by gorzelinski — COMMENTED** [review]
Ideas for some minor tweaks.

File: docs/getting-started/index.md [hunk 3]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
 
 # Getting started with CKEditor&nbsp;5
 
-CKEditor&nbsp;5 provides every type of WYSIWYG editing solution imaginable. From editors similar to Google Docs and Medium to Slack or Twitter-like applications, all is possible within a single editing framework. It is an ultra-modern JavaScript rich-text editor with MVC architecture, custom data model, and virtual DOM, written from scratch in TypeScript with excellent webpack and Vite support. Find out the most convenient way to start using it!
+CKEditor&nbsp;5 is a flexible editing framework, which provides every type of WYSIWYG editing solution imaginable. From editors similar to Google Docs and Medium to Notion, Slack or Twitter-like applications, all is possible within a single editing framework. It is an ultra-modern JavaScript rich-text editor with MVC architecture, custom data model, and virtual DOM, written from scratch in TypeScript with excellent webpack and Vite support. Find out the most convenient way to start using it!
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/index.md:11]: In commit.

File: docs/getting-started/index.md [hunk 4]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
...
 	<info-box>
 		Use the **main menu button in the upper-left corner** to navigate through the documentation.
 	</info-box>
 </span>
 
-## New to CKEditor&nbsp;5?
+## Are you new to CKEditor&nbsp;5?
 
-Find out how to start your adventure with the {@link getting-started/quick-start Quick Start} guide.
+If your dive into using our WYSIWYG editor is only starting, find out how to kick off this adventure easily with the {@link getting-started/quick-start Quick Start} guide. You will learn how to run CKEditor&nbsp;5 from npm or using CDN.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/index.md:21]: In commit.

File: docs/getting-started/index.md [hunk 5]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
...
-
-## CKEditor&nbsp;5 integrations
+## CKEditor&nbsp;5 framework integrations
 
-CKEditor&nbsp;5 is framework agnostic and can be integrated with any JavaScript framework. CKEditor&nbsp;5 is a native JavaScript rich-text editing component written in TypeScript. It does not require any uncommon techniques or technologies to be used. Therefore, unless the framework you use has atypical limitations, CKEditor&nbsp;5 is compatible with it. Native integrations with the most popular libraries will save you time and effort.
-
-### Official WYSIWYG editor integrations
-
-There are four official integrations so far:
+Dou you prefer to use ready-made frameworks? Native integrations with the most popular libraries will save you time and effort. There are four official integrations so far:
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/index.md:29]: Dost thou ;)

File: docs/getting-started/index.md [hunk 6]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
...
 
 * {@link getting-started/integrations/angular CKEditor&nbsp;5 rich-text editor for Angular}
 * {@link getting-started/integrations/react CKEditor&nbsp;5 rich-text editor for React}
 * {@link getting-started/integrations/vuejs-v2 CKEditor&nbsp;5 rich-text editor for Vue.js 2.x}
 * {@link getting-started/integrations/vuejs-v3 CKEditor&nbsp;5 rich-text editor for Vue.js 3.x}
 
-Refer to their documentation to learn how to use them.
+However, some more frameworks are also supported. Refer to their documentation on the left to learn how to use them. CKEditor&nbsp;5 is a native JavaScript rich-text editing component written in TypeScript. As such, it is framework agnostic and can be integrated with any JavaScript framework. It does not require any uncommon techniques or technologies to be used. Therefore, unless the framework you use has atypical limitations, CKEditor&nbsp;5 is compatible with it.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/index.md:36]: That's actually true and we have had this on out website for like year or so?

File: docs/getting-started/index.md [hunk 7]
```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
...
+* If none exists, integrate CKEditor&nbsp;5 with your framework by yourself.
 
-	There are four official integrations now: for {@link getting-started/integrations/react React}, {@link getting-started/integrations/angular Angular}, {@link getting-started/integrations/vuejs-v2 Vue.js 2.x}, and {@link getting-started/integrations/vuejs-v3 Vue.js 3.x}.
-2. **If not, search for community-driven integrations.** Most of them are available on [npm](https://www.npmjs.com/).
-3. **If none exists, integrate CKEditor&nbsp;5 with your framework by yourself.**
+CKEditor&nbsp;5 offers {@link getting-started/legacy-getting-started/predefined-builds predefined builds} that expose a rich JavaScript API, which you can use to {@link getting-started/legacy-getting-started/editor-lifecycle#creating-an-editor-with-create create editors} and {@link getting-started/getting-and-setting-data#setting-the-editor-data-with-setdata control them}. -->
+
+## Configuring CKEditor&nbsp;5
+
+Once you install your copy of CKEditor&nbsp;5, take some time to {@link getting-started/setup/configuration configure it} before first use. Set up data saving methods, editor toolbars and UI.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/index.md:55]: In commit.

File: docs/getting-started/index.md (hunk 2 above)
Comment (godai78) [reply docs/getting-started/index.md:59]: It's like first point of contact, that's why we had it on top for the past monts (and still have). I judged we can keep it, just not that exposed as before.

File: docs/getting-started/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (gorzelinski) [reply docs/getting-started/index.md]: It appears so ;)

**Pull request merged**